tasks/unmute.js

```diff
@@ -0,0 +1,16 @@
+/*
+
+	This is to be used with the mute command located in
+	/commands/Moderation/mute.js
+
+*/
+const { Task } = require('klasa');
+
+module.exports = class extends Task {
+
+	async run({ guild, user }) {
+		const _guild = this.client.guilds.get(guild);
+		await _guild.members.fetch(user).roles.remove(_guild.configs.roles.muted);
```

Comment:
fetch is a Promise, Promise doesn't have a static property called roles.

I suggest
```js
if (!_guild) return;
const member = await _guild.members.fetch(user).catch(() => null);
if (!member) return;
await member.roles.remove(_guild.configs.roles.muted);
```